content/docs/develop/getting-started/addon-basics.md

```diff
@@ -6,18 +6,18 @@ aliases:
 ---
 
 ## What is an addon?
-An addon is a userscript, userstyle, or both that runs on the Scratch website or project editor when enabled. For example, the "Editor find bar" addon has a userscript that adds a find box to the editor, and a userstyle that styles to that box.
+An addon is a userscript, userstyle, or combination of both that runs on the Scratch website or project editor when enabled. For example, the "Editor find bar" addon has a userscript that adds a find box to the editor, and a userstyle that styles to that box.

```

Comment:
A good opportunity to change "styles to that box" to "styles that box"
Maybe I'm wrong